index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Microverse Project- Survey form: Christian-Jesse</title>
+  <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+  <div class="container">
+
+<header class="header">
+  <h1 id="title" class="text-center"> Survey Form</h1>
+  <p id="description" class="description text-center">
+          Thank you for taking time for this<p>
+    </header>
+  <form id="survey-form">
+    <div class="form-group">
+      <label id="name-label" for="name"> My Name</label>
+      <input type="text" name="name" id="name" class="form-control" placeholder="Enter your name" required="">
+    </div>
+
+    <div class="form-group">
+      <label id="email-label" for="email">My Email</label>
+      <input type="email" name="email" id="email" class="form-control" placeholder="Enter your Email" required="">
+    </div>
+
+    <div class="form-group">
+      <label id="number-label" for="number">My Age</label>
+      <input type="number" name="age" id="number" min="5" max="70" class="form-control" placeholder="Age">
+    </div>
+
+    <div class="form-group">
+      <p>What is your current role?</p>
+      <select id="dropdown" name="role" class="form-control" required="">
+        <option disabled="" selected="" value="">Select current role</option>
+        <option value="student">Student</option>
+        <option value="job">Part Time Job</option>
+        <option value="learner">Other</option>
+      </select>
```

Comment:
Too many ID's used here. It would be better using class selector instead of ID because class selector can help you organize well you code and it can also be used by a JavaScript (via the HTML DOM) to make changes to HTML elements with a specified class.